network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/text_only_teaser_block.html

```diff
@@ -2,23 +2,21 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block block_content %}
-  <div class="tw-row tw-py-4">
-	{% for block in self.cards %}
-      {% with card=block %}
-        <div class="tw-px-4 medium:tw-w-1/3 tw-mb-5">
-          <div class="tw-bg-white tw-relative tw-py-4 tw-w-full tw-flex tw-flex-col">
-            {% if card.heading_link %}<a href="{% pageurl card.heading_link %}">{% endif %}
-            <span class="tw-h5-heading tw-mb-2">{{ card.heading }}</span>
+  <section class="tw-container tw-mb-7">
+    <div class="tw-row tw-grid medium:tw-grid-cols-3 tw-gap-x-4 tw-gap-y-5 medium:tw-gap-6">
+      {% for block in self.cards %}
+        {% with card=block %}
+          <div class="tw-pt-4 tw-relative tw-w-full tw-flex tw-flex-col tw-border-t tw-border-black/10">
```

Comment:
What is this `tw-border-black/10` thing? Is that a new Tailwind thing I missed?

Reply:
This syntax has actually been in since 2.0 I think.  😅  Basically it allows you to use colour opacity shorthand any time you use a colour on a class. 

Extra info for fun:
Tailwind uses rgba for all their colours (even when you have a hex in the config it converts) and they have this opacity css var that it hooks into for example:
<img width="510" alt="Screen Shot 2022-10-28 at 8 58 20 AM" src="https://user-images.githubusercontent.com/25041665/198664361-f3811afd-40fd-441e-97aa-f94470752691.png">

So when this border class is generated it looks like:
<img width="288" alt="Screen Shot 2022-10-28 at 8 58 37 AM" src="https://user-images.githubusercontent.com/25041665/198664616-306adacd-fc42-4f52-a56e-f7d3275684ab.png">

Most properties modifying colour will have it documented on their own page eg.
https://tailwindcss.com/docs/text-color#changing-the-opacity

Reply:
Thanks @stevedya. TIL. I did not know about this one at all.